test/lit/binary/empty-param-name.test

```diff
@@ -0,0 +1,6 @@
+;; Test that we show a warning on an empty local name, and do not crash.
```

Comment:
- Would be good to include the contents of the wasm file
- I think this depends on the wabt's behavior that emits an empty local name? Would it be good to reproduce the behavior from our end?

Reply:
Hmm, I consider wabt's behavior to be a bug. Maybe not literally, it's valid with the spec, but an empty name is just worse for debugging than an index. So it's better to not emit it. I opened a wabt issue meanwhile: https://github.com/WebAssembly/wabt/issues/1799

But I guess it's debatable. I'm ok with removing the warning (and skipping such names so we don't crash).

Reply:
(added content details to the testcase as requested, thanks)

Reply:
If it's a bug from the wabt side, isn't this test relying on that bug? Would this test still work after that bug is fixed?

Reply:
Yes, if wabt fixes the bug then recreating the wasm won't cause a warning. Still, I think it's good to test that we don't crash + that we warn on an invalid input in a clear way. I don't feel strongly though.

I also am ok with either warning or not warning.

Reply:
Ah right, we still have the binary.. I was confused and thought if the wabt bug is fixed this test would no longer work, sorry. This sounds OK, especially given that the test file description says the file was generated from the bug.